Doc/whatsnew/3.15.rst

```diff
@@ -154,6 +154,10 @@ that may require changes to your code.
 Build changes
 =============
 
+* Removed bundled copy of the libmpdec, use system library if it's
+  available.
```

Comment:
What's New should ideally describe *why* we've done something, not just *what* we've done.
```suggestion
* Remove the bundled libmpdec_ decimal library from the CPython source tree,
  to simplify maintenence and updates. The :mod:`decimal` module will now
  unconditionally use the system's libmpdec decimal library. Also remove the
  now unused :option:`!--with-system-libmpdec` :program:`configure` flag.
  
  .. _libmpdec: https://www.bytereef.org/mpdecimal/
```